modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,197 @@
+
+Logstash, Insight and InfluxDB are all K8s stateful sets.
+
+
+=== Monitoring Center Components
+
+Monitoring Center components represents the "Anypoint Monitoring" product that visualizer the data stored in DIAS products.
+
+All user traffic goes to the experience api (`monitoring-center-ui-api`) and from there it can proxied to:
+
+- `monitoring-center-settings-api` to manage user settings.
+
+- `monitoring-center-alerts-api` to manage alerts, synchronize the changes with Insights engine and trigger basic alerts notifications.
+
+- `monitoring-center-visualizer` that is Grafana fork. It manages the dashboards logic, advanced alerts and the rest of the functionality available in PCE 3.0.
+
+- `monitoring-center-metrics-api` to validate Influx DB queries and proxy the calls to the data source (`dias-prov-k8s-am-influxdb-comp`).
+
+=== Visualizer Components
+
+Visualizer components represents the "Anypoint Visualizer" product.
+
+Logstash `dias-prov-k8s-am-ingestor-comp` sends a subset of mule agent events to the `visualizer-topology-processor` service.
+That service process those events and stores the results in the Postgres DB (`stolon-amv`).
+The final application network generation will be handled by `visualizer-topology-api` service and is invoked by a user traffic.
+There is a set of cron jobs to prevent the uncontrolled DB growth for mule agent events. They perform the stale data clean and runs as part of `visualizer-janitor` service.
+
+All user traffic comes to the `visualizer-experience-api` service that orchestrates loading all necessary data from different data sources (like `visualizer-topology-api`, `monitoring-center-ui-api`, etc.) and performs further data decorations.
+
+== Considerations
+
+=== Cluster secrets
+
+All Anypoint Monitoring/Visualizer components use self-signed certificates for TLS support. Those certificates are generated by the `dias-prov-k8s-amr-certs-comp` components and stored as secrets within the cluster under the `dias` namespace. It is important to notice if the customer changes the platform DNS, the `dias-prov-k8s-amr-certs-comp` will be redeployed and secrets will be updated. Additionally, the `dias-prov-k8s-amr-ingestor-router-comp` component, running Nginx, will be redeployed as well, refreshing those new secrets.
+
+=== Stateful sets directories
+
+Each stateful set uses different directories to hold its data, resulting in the following directories:
+
+```
+  /var/lib/data/influxdb/data
+  /var/lib/data/influxdb/meta
+  /var/lib/data/logstash
+  /var/lib/data/dias-cassandra
+  /var/lib/data/dias-meld
+```
+
+InfluxDB meta pods (3 pods) use `/var/lib/data/influxdb/meta` while data pods (2 pods) use `/var/lib/data/influxdb/data`. Logstash uses `/var/lib/data/logstash` and finally Insight uses the remaining two directories to hold its data.
+
+== Requirements
```

Comment:
No need to repeat hardware requirements. They should all be consolidated in the PCE 3.0 Hardware Requirementes section

Reply:
Done. Left the port section, though